editoast/migrations/2025-11-07-085502_etcs_level2_from_etcs_brake_params/up.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE rolling_stock
+ADD COLUMN supported_signaling_systems_tmp jsonb;
+
+UPDATE rolling_stock
+SET supported_signaling_systems_tmp = (
+    SELECT jsonb_agg(jsonb_build_object('type', value::text))
+    FROM unnest(supported_signaling_systems) AS value
+);
```

Comment:
When doing this, if I understand correctly, you will get the value `ETCS_LEVEL2` into `supported_signaling_systems_tmp`... but without the `brake_params`. Can you test that? Should we filter out this value in here and let the lines below properly filling the `ETCS_LEVEL2` case?